apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.html.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { QuestionsTable } from '../../components/QuestionsTable.html';
+
+export const QuestionsPage = ({ questions, resLocals }) => {
+  const showAddQuestionButton = resLocals.authz_data.has_course_permission_edit &&
+  !resLocals.course.example_course &&
+  !resLocals.needToSync;
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../pages/partials/head') %>", resLocals)}
+      </head>
+
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+
+        <form class="ml-1 btn-group" name="add-question-form" method="POST">
+          <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+          <input type="hidden" name="__action" value="add_question" />
+        </form>
```

Comment:
I think its a bit weird to have this form here when I am passing a 'showAddQuestionButton' parameter to the QuestionsTable component. But I also thought it felt weird to pass the CSRF token to the component. Let me know what you think is best.

Reply:
Passing a CSRF token to a component is totally valid!

Reply:
fixed, thanks